examples/elixir/e10.py

```diff
@@ -10,39 +10,37 @@
 import logging
 
 from term import Atom
-from pyrlang.gen_server import GenServer
+from pyrlang.gen.server import GenServer
+from pyrlang.gen.decorators import call, cast, info
 from pyrlang import Node
-from pyrlang import GeventEngine as Engine
-# from pyrlang import AsyncioEngine as async
 
 LOG = logging.getLogger("+++EXAMPLE10+++")
 logging.getLogger("").setLevel(logging.DEBUG)
 
 
 class MyProcess(GenServer):
     def __init__(self, node) -> None:
-        GenServer.__init__(self, node.node_name_,
-                           accepted_calls=['hello', 'hello_again'])
+        GenServer.__init__(self)
         node.register_name(self, Atom('my_process'))
         LOG.info("registering process - 'my_process'")
 
-    def hello(self):
+    @call(1, lambda msg: True)
```

Comment:
the number is to reflect the order that the pattern will be evaluated with. the function is if it match or not. one of the calls wil end up first and will always match. hence we have no way of controlling when which function is called.

Think what you want is
```
@call(1, lambda msg: msg == "hello")
```

Reply:
thanks! I didn't actually know what those were doing.